source/fundamentals/authentication/enterprise-mechanisms.txt

```diff
@@ -21,10 +21,19 @@ parameters of the
 :manual:`connection string </reference/connection-string/>`:
 
 - Set the ``authMechanism`` parameter to ``GSSAPI``
-- Set the ``gssapiServiceName`` if using a value other than ``mongodb``
+- Set the ``SERVICE_NAME`` value in the ``authMechanismProperties``
+  parameter if using a value other than ``mongodb``
 - Specify a ``SERVICE_REALM`` value in the ``authMechanismProperties``
   parameter if a custom service realm is required.
 
+.. note::
+
+   The ``gssapiServiceName`` parameter, although still supported by the MongoDB Node driver,
+   is a *deprecated* alias for ``authMechanismProperties=SERVICE_NAME:<your service name>``.
+   See the
+   :manual:`authMechanismProperties </reference/connection-string/#urioption.authMechanismProperties>`
+   parameter documentation for more information.
+
```

Comment:
```suggestion
.. important::

   The ``gssapiServiceName`` parameter is deprecated and may be removed in future versions of the driver. Use ``authMechanismProperties=SERVICE_NAME:<your service name>`` in the connection URI instead.
   See the
   :manual:`authMechanismProperties </reference/connection-string/#urioption.authMechanismProperties>`
   parameter documentation for more information.

```

Reply:
One suggestion to tighten this up and make it more active and prescriptive.

Reply:
Thanks for the note. I agree that this prescriptive and direct phrasing is more helpful to users in that it tells them which decision to make. Thanks for drawing my attention to the "important" directive as well, have not used one yet and didn't occur to me (but deprecation warnings are important). Will try to keep these edits in mind going forward.

Fix: https://github.com/mongodb/docs-node/pull/147/commits/a43f7461c88ff0e514f0c02f32cc573cfa673c4a